ISSUE-240:Turning off Reverse DNS lookup

raghavf1 opened the pull request:
Reverse DNS is used to lookup the hostname of the IP address where the request is originating from. This is typically used for logging purposes, so domain names can be shown instead of IP addresses. However, this doesn't make sense in cloud and/or cluster deployments, since the originating IP address will be an internal address, e.g. the address of the load balancer etc. Since doing a reverse DNS will not provide any value, it will result in unneeded overhead.
 Reverse DNS lookups have also been shown to be a source of spikes, in case the DNS server / resolver get into temporarily non-responsive.
Regarding backward compatibility: Since its main purpose is logging, there shoudn't be any b/w compatibility issues.

MadhuriArugula commented:
@raghavf1 Can you update the PR with an explanation of the change and that it will not impact backwards compatibility?
Also you might want to fix the unit tests failed because of this change.

raghavf1 commented:
> @raghavf1 Can you update the PR with an explanation of the change and that it will not impact backwards compatibility?
> Also you might want to fix the unit tests failed because of this change.

@MadhuriArugula I have updated description of the PR and fixed the corresponding tests. Please let me know if you need any further info

MadhuriArugula commented on charts/pega/config/deploy/prconfig.xml:
```diff
@@ -7,6 +7,7 @@
         <env name="database/databases/PegaDATA/dataSource" value="java:comp/env/jdbc/PegaRULES"/>
         <env name="security/urlaccesslog" value="NORMAL" />
         <env name="security/urlaccessmode" value="WARN" />
+        <env name="http/reversednslookup" value="false" />​
```
@raghavf1 Is it mandatory to set reverse DNS lookup to` false `by default? or can this be configurable from values.yaml? 
As per the user story and bug item mentioned in #240, it is under Engine backlog. 

Can you confirm if this was verified with Engine teams? Some approvals from Engine can help us proceeding with merge.

kishorv10 commented:
Since there has been no activity on this pull request for the past year, I will proceed to close it. Please feel free to reopen it if necessary.
